config/global-repos.list

```diff
@@ -0,0 +1,20 @@
+blindnet-io/product-management
+blindnet-io/clients-management
+blindnet-io/communication-management
+blindnet-io/openness-framework
+blindnet-io/devrel-management
+blindnet-io/product-ideas
+blindnet-io/privateform-frontend
+blindnet-io/privateform-backend
+blindnet-io/blindnet-docs
+blindnet-io/blindnet-backend-scala
+blindnet-io/blindnet-backend
+blindnet-io/blindnet-sdk-java
+blindnet-io/blindnet-sdk-java-server
+blindnet-io/dotnet-sdk
+blindnet-io/blindnet-sdk-js
+blindnet-io/blindnet-sdk-php
+blindnet-io/blindsend-java-lib
+blindnet-io/blindsend-fe
+blindnet-io/blindsend-be
+blindnet-io/blindsend
```

Comment:
Add the following ones too:

- blindnet-io/gateway
- blindnet-io/dashboard-front
- blindnet-io/identity
- blindnet-io/sdk-js-lite
- blindnet-io/blindnet-sdk-node
- blindnet-io/blindnet-sdk-dotnet-core-server
- blindnet-io/blindnet-sdk-dotnet-framework-server
- blindnet-io/blindsend-server